tools/integrations/manage.sh

```diff
@@ -217,26 +224,61 @@ cmd_publish() {
   if [[ "airbyte/normalization" == "${image_name}" ]]; then
     echo "Publishing normalization images (version: $versioned_image)"
     GIT_REVISION=$(git rev-parse HEAD)
-    VERSION=$image_version GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml build
-    VERSION=$image_version GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml push
-    VERSION=latest         GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml build
-    VERSION=latest         GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml push
+
+    # Note: "buildx bake" needs to be run within the directory
+    local original_pwd=$PWD
+    cd airbyte-integrations/bases/base-normalization
+
+    VERSION=$image_version GIT_REVISION=$GIT_REVISION docker buildx bake \
+      --set "*.platform=$build_arch"                                     \
+      -f docker-compose.build.yaml                                       \
+      --push
+
+    VERSION=latest GIT_REVISION=$GIT_REVISION docker buildx bake \
+      --set "*.platform=$build_arch"                             \
+      -f docker-compose.build.yaml                               \
+      --push
+
+    cd $original_pwd
   else
-    docker tag "$image_name:dev" "$versioned_image"
-    docker tag "$image_name:dev" "$latest_image"
+    # We have to go arch-by-arch locally (see https://github.com/docker/buildx/issues/59 for more info) due to our base images (e.g. airbyte-integrations/bases/base-java)
+    # Alternative local approach @ https://github.com/docker/buildx/issues/301#issuecomment-755164475
+
+    for arch in $(echo $build_arch | sed "s/,/ /g")
+    do
+      echo "building base images for $arch"
+      docker buildx build -t airbyte/integration-base:dev --platform $arch --load airbyte-integrations/bases/base
+      docker buildx build -t airbyte/integration-base-java:dev --platform $arch --load airbyte-integrations/bases/base-java
+
+      local arch_versioned_image=$image_name:`echo $arch | sed "s/\//-/g"`-$image_version
+      echo "Publishing new version ($arch_versioned_image) from $path"
+      docker buildx build -t $arch_versioned_image --platform $arch --push $path
+      docker manifest create $latest_image --amend $arch_versioned_image
+      docker manifest create $versioned_image --amend $arch_versioned_image
+    done
+
+    docker manifest push $latest_image
+    docker manifest push $versioned_image
+    docker manifest rm $latest_image
```

Comment:
nit: can we leave a comment here explaining why this rm is needed?

Reply:
It's actually not needed for the production version of this script, but if you are me, testing this locally repeatedly, you'll keep adding to your existing local manifest file.. so your "latests" manifest will just keep gaining tags if you don't `rm` each time to start fresh.